### PR TITLE
Add containerstore to autofill exception on macos

### PR DESCRIPTION
**Pull request by GioSensation**
This site has multiple autofill issues. See [Asana](https://app.asana.com/0/1200930669568058/1202105131841960/f) for context.

### CODE REVIEW

**Comment by GioSensation** [comment]
@jonathanKingston I've added this exception for autofill on macOS. Is this the right way to do it?

File: overrides/macos-override.json [hunk 1]
```diff
@@ -31,7 +31,13 @@
             "state": "enabled"
         },
         "autofill": {
-            "state": "enabled"
+            "state": "enabled",
+            "exceptions": [
+                {
+                    "domain": "containerstore.com",
+                    "reason": "Poor UX: https://app.asana.com/0/1200930669568058/1202105131841960/f"
```
Comment (SlayterDev) [inline overrides/macos-override.json:38]: Let's keep Asana links out of the config. There is potential one day in the future these reasons could be user facing.

**Review by SlayterDev — CHANGES_REQUESTED** [review]

File: overrides/macos-override.json (hunk 1 above)
Comment (GioSensation) [reply overrides/macos-override.json]: Done!

**Review by SlayterDev — APPROVED** [review]
LGTM 👍

**Pull request merged**